File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag25e26/ExercicioC.java
package manzanoPag25e26;

import java.util.Scanner;
import java.lang.Math;

/*c) Calcular e apresentar o valor do volume de uma lata de óleo, utilizando a fórmula: volume<- pi*raio sobre 2 * Altura */


public class ExercicioC {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int raio, altura;
		float volume;
		
		System.out.println("Digita o valor do raio:");
		raio = sc.nextInt();
		System.out.println("Digita o valor da altura:");
		altura = sc.nextInt();
				
				 
		raio = (int) Math.pow(raio, 2);
		
		 volume = 3.14f * raio * altura;

		 System.out.println("O volume da lata de óleo é: "+volume);
		
		sc.close();

	}

}

File: LogicaDeProgramacao(JavaScript)/exercicios pag 25 e 26(Manzano)/js/exercicioC.js
/*c) Calcular e apresentar o valor do volume de uma lata de óleo, utilizando a fórmula: volume<- pi*raio sobre 2 * Altura */

var raio = parseInt(prompt("Digita o valor do raio:"))
var altura = parseInt(prompt("Digita o valor da altura:"))
 

var volume = 3.14 * raio**2 * altura

alert("O volume da lata de óleo é: "+volume)
